src/components/book/BookContentCardDetail.js

```diff
@@ -118,13 +126,12 @@ class BookContentCardDetail extends Component {
 
     return (
       <Fragment>
-        <MyButton
-          tip="Learn more"
+        <CardMedia
+          className={classes.media}
+          image={this.props.bookImage}
+          title={this.props.title}
           onClick={this.handleOpen}
-          btnClassName={classes.infoButton}
-        >
-          <InfoIcon />
```

Comment:
On line 154 for your code you wrote "diviers"

I believe that should be "dividers"?

Reply:
I removed it